doc/api/documentation.md

```diff
@@ -66,6 +66,10 @@ used. Bugs or behavior changes may surprise users when Experimental API
 modifications occur. To avoid surprises, use of an Experimental feature may need
 a command-line flag. Experimental features may also emit a [warning][].
 
+An experimental global API in early development and active development must need
```

Comment:
I think it really depends on the feature/reasoning and it may be better to require TSC approval for adding globals (which I somehow misremembered that we required already).

I think _sometimes_ a command line flag may be sufficient and sometimes it's too much/not enough depending on the specific experiment.

I think sometimes we may choose to require a build time flag, a runtime flag + warning, just a runtime flag, just a warning or nothing at all but documenting the API as experimental/active-development.

Reply:
I think for an _unfinished_ experimental feature, either a build time flag, a runtime flag, or a warning should be applied. Just that it should not be enabled unconditionally without any of these facilities.